src/evergreen.ts

```diff
@@ -80,7 +108,18 @@ export class client {
             username: username,
             password: password,
         };
-        this.postResource(callback, "login", params);
+        this.postUIResource(callback, "login", params);
+    }
+
+    /**
+     * Gets patches for a particular user
+     * @param callback - function to process the response
+     * @param username - Evergreen user's username
+     * @returns nothing
+     */
+    public getPatches(callback: request.RequestCallback, username?: string) {
+        const resource =  "json/patches/user/" + username;
+        this.getUIResource(callback, resource, {});
```

Comment:
"params" is marked as an optional parameter, so you don't need to pass an empty object